City-of-Pori/pori-kada
# Pori-55: Project liftup tweaks

Project liftup style tweaks. Wrapped visible title field in h3. Changed postdate format.

To test:
1. drush fra -y && drush cc all
2. have two liftups with project flags promoted to frontpage
3. confirm that liftup title is h3
4. confirm liftup styles look good and are responsive.
